packages/components/e2e/component-objects/src/main/java/org/talend/component/list/table/Table.java

```diff
@@ -0,0 +1,117 @@
+package org.talend.component.list.table;
+
+import org.talend.component.Component;
+import org.openqa.selenium.By;
+import org.openqa.selenium.NotFoundException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+
+import static java.util.stream.Collectors.toList;
+
+/**
+ * A List is used to easy access to WebElements of the react-talend-component List component - Table.
+ */
+public class Table extends Component {
+
+    private static final String NAME = "Table";
+
+    private static final String TABLE_SELECTOR = ".tc-list-table%s";
+
+    private static final String TABLE_COLUMN_HEADER_SELECTOR = ".ReactVirtualized__Table__headerColumn";
+
+    private static final String TABLE_COLUMN_HEADER_KEY_CLASS = TABLE_COLUMN_HEADER_SELECTOR + ".tc-list-cell-%s";
+
+    private static final String TABLE_ITEM_SELECTOR = ".ReactVirtualized__Table__row";
+
+    /**
+     * Constructor.
+     *
+     * @param driver Selenium WebDriver
+     */
+    public Table(final WebDriver driver) {
+        this(driver, null);
+    }
+
+    /**
+     * Constructor with table id
+     *
+     * @param driver Selenium WebDriver
+     */
+    public Table(final WebDriver driver, final String id) {
+        super(driver, NAME, String.format(TABLE_SELECTOR, id != null ? "#" + id : ""));
```

Comment:
I can't, no default value in methode prototype in java.
Plus, super() must be the first instruction. So no choice :(